docs/guide.mdx

```diff
@@ -119,12 +119,38 @@ it'll work just fine.
 
 ### Languages & Frameworks
 
-Sapphire is programmable using any language that targets the EVM, such as Solidity
-or Vyper. If you prefer to use an Ethereum framework like Hardhat or Foundry, you
-can also use those with Sapphire; all you need to do is set your Web3 gateway URL.
+Sapphire is programmable using any language that targets the EVM, such as Solidity,
+Fe or Vyper. If you prefer to use an Ethereum framework like Hardhat or Foundry,
+you can also use those with Sapphire; all you need to do is set your Web3 gateway URL.
 You can find the details of the Oasis Sapphire Web3 endpoints
 [here](https://github.com/oasisprotocol/docs/blob/main/docs/dapp/sapphire/README.mdx#rpc-endpoints).
 
+
+#### Wagmi
```

Comment:
This chapter is more about writing contracts, deploying and managing them. And it is already getting pretty long. Wagmi is a react-based framework right? I would move it to the [Browser](https://deploy-preview-292--oasisprotocol-sapphire-paratime.netlify.app/dapp/sapphire/browser) chapter instead and split the chapter into *Hardhat* (most of the existing content) and *Wagmi* (this section).